lib/ExpensiMark.js

```diff
@@ -503,6 +518,23 @@ export default class ExpensiMark {
                     return `!(${g2})`;
                 },
             },
+
+            {
+                name: 'video',
+                regex: /<video[^><]*data-expensify-source\s*=\s*(['"])(\S*?)\1(.*?)>([^><]*)<\/video>*(?![^<][\s\S]*?(<\/pre>|<\/code>))/gi,
+                replacement: (match, g1, g2, g3, g4, ...args) => {
+                    const extras = args[args.length - 1];
+                    if (g3 && extras && extras.cacheVideoAttributes && typeof extras.cacheVideoAttributes === 'function') {
+                        extras.cacheVideoAttributes(g2, g3); // cacheVideoAttributes(source, attributes)
+                    }
+                    if (g4) {
```

Comment:
Could you add some comments here and below to describe better what `g4` and `g2` is? It might be best to just give them actual names?